app/web/css/components/modal.scss

```diff
@@ -0,0 +1,50 @@
+@import '../helpers/breakpoints';
+@import '../helpers/mixins';
+@import '../helpers/variables';
+
+// show the modal when the input is checked
+input[type="checkbox"]:checked.modal + label + .modal__value,
```

Comment:
Be aware that by using + operators you're coupling the css to html. Also prevent the use of html elements (br). If you see fit to prevent that please do! This applies to all scss assets if applicable.

Reply:
Good one.  I actually did this for the tooltip, but forgot to remove it for the modal.  Greatly simplifies the css code as well :-)  💯